example/pubspec.yaml

```diff
@@ -17,7 +17,6 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_dotenv: ^5.0.2
```

Comment:
removing it because we will be adding a separate sample app in the same repo to demonstrate this

Reply:
Thanks for the comment. I was going to ask why this change was made.